lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -216,8 +217,36 @@ module.exports = {
     return eventAuthorizers;
   },
 
-  validateAlbAuth(auth) {
-    auth.onUnauthenticatedRequest = auth.allowUnauthenticated ? 'allow' : 'deny';
+  validateAlbAuth(auth, authName) {
+    const hasAllowUnauthenticated = _.has(auth, ['allowUnauthenticated']);
```

Comment:
Let's not use lodash for constructs which can easily be written in plain JS (see https://github.com/serverless/serverless/issues/7747)